exercises/poker/canonical-data.json

```diff
@@ -0,0 +1,317 @@
+      "property": "bestHands",
+      "input": [
+        "4S 5H 4C 8D 4H",
+        "10D JH QS KD AC"
+      ],
+      "expected": [
+        "10D JH QS KD AC"
+      ]
+    },
+    {
+      "description": "aces can start a straight (A 2 3 4 5)",
+      "property": "bestHands",
+      "input": [
+        "4S 5H 4C 8D 4H",
+        "4D AH 3S 2D 5C"
+      ],
+      "expected": [
+        "4D AH 3S 2D 5C"
+      ]
+    },
+    {
+      "description": "both hands with a straight, tie goes to highest ranked card",
+      "property": "bestHands",
+      "input": [
+        "4S 6C 7S 8D 5H",
+        "5S 7H 8S 9D 6H"
+      ],
+      "expected": [
+        "5S 7H 8S 9D 6H"
+      ]
+    },
+    {
+      "description": "even though an ace is usually high, a 5-high straight is the lowest-scoring straight",
+      "property": "bestHands",
+      "input": [
+        "2H 3C 4D 5D 6H",
+        "4S AH 3S 2D 5H"
+      ],
+      "expected": [
+        "2H 3C 4D 5D 6H"
+      ]
+    },
+    {
+      "description": "flush beats a straight",
+      "property": "bestHands",
+      "input": [
+        "4C 6H 7D 8D 5H",
+        "2S 4S 5S 6S 7S"
+      ],
+      "expected": [
+        "2S 4S 5S 6S 7S"
+      ]
+    },
+    {
+      "description": "both hands have a flush, tie goes to high card, down to the last one if necessary",
+      "property": "bestHands",
+      "input": [
+        "4H 7H 8H 9H 6H",
+        "2S 4S 5S 6S 7S"
+      ],
+      "expected": [
+        "4H 7H 8H 9H 6H"
+      ]
+    },
+    {
+      "description": "full house beats a flush",
+      "property": "bestHands",
+      "input": [
+        "3H 6H 7H 8H 5H",
+        "4S 5H 4C 5D 4H"
+      ],
+      "expected": [
+        "4S 5H 4C 5D 4H"
+      ]
+    },
+    {
+      "description": "both hands have a full house, tie goes to highest-ranked triplet",
+      "property": "bestHands",
+      "input": [
+        "4H 4S 4D 9S 9D",
+        "5H 5S 5D 8S 8D"
+      ],
+      "expected": [
+        "5H 5S 5D 8S 8D"
+      ]
+    },
+    {
+      "description": "with multiple decks, both hands have a full house with the same triplet, tie goes to the pair",
+      "property": "bestHands",
+      "input": [
+        "5H 5S 5D 9S 9D",
+        "5H 5S 5D 8S 8D"
+      ],
+      "expected": [
+        "5H 5S 5D 9S 9D"
+      ]
+    },
+    {
+      "description": "four of a kind beats a full house",
+      "property": "bestHands",
+      "input": [
+        "4S 5H 4D 5D 4H",
+        "3S 3H 2S 3D 3C"
+      ],
+      "expected": [
+        "3S 3H 2S 3D 3C"
+      ]
+    },
+    {
+      "description": "both hands have four of a kind, tie goes to high quad",
+      "property": "bestHands",
+      "input": [
+        "2S 2H 2C 8D 2D",
+        "4S 5H 5S 5D 5C"
+      ],
+      "expected": [
+        "4S 5H 5S 5D 5C"
+      ]
+    },
+    {
+      "description": "with multiple decks, both hands with identical four of a kind, tie determined by kicker" ,
```

Comment:
> the bowling test data contains words like Turkey, Double, etc...

errr? https://github.com/search?q=org%3Aexercism+turkey&type=Code

and just in case that missed something, nothing in https://github.com/exercism/x-common/blob/master/exercises/bowling/canonical-data.json either.

???

confused

---

Do you know though, in this case the problem is solved: "kicker" is defined earlier https://github.com/exercism/x-common/pull/793/files#r117595066